src/sanitizer.js

```diff
@@ -37,14 +37,24 @@ const BLACKLISTED_TAGS = {
   'object': true,
   'script': true,
   'style': true,
+  // SVG is blacklisted temporarily. There's no intention to keep this block
```

Comment:
This whole file will be replaced with the AMP Validator implementation. See https://github.com/ampproject/amphtml/issues/1156. I will make a note here.